test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -0,0 +1,335 @@
+            $output = ConvertFrom-Markdown -Path $mdFile.FullName -AsVT100EncodedString
+            $output.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+
+        It 'Can convert input from a fileinfo object to vt100 encoded string' {
+            $ouputFromFileInfo = $mdFile | ConvertFrom-Markdown -AsVT100EncodedString
+            $ouputFromFileInfo.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+
+        It 'Can convert input from a literal path to vt100 encoded string' {
+            $output = ConvertFrom-Markdown -Path $mdLiteralPath -AsVT100EncodedString
+            $output.VT100EncodedString | Should -BeExactly $expectedStringFromFile
+        }
+    }
+
+    Context "ConvertFrom-Markdown error cases" {
+        It "Gets an error if path is not FileSystem provider path" {
+            { ConvertFrom-Markdown -Path Env:\PSModulePath -ErrorAction Stop } | Should -Throw -ErrorId 'OnlyFileSystemPathsSupported,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+
+        It "Gets an error if path does not exist" {
+            { ConvertFrom-Markdown -Path DoestnotExist -ErrorAction Stop } | Should -Throw -ErrorId 'FileNotFound,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+
+        It "Gets an error if input object type is not correct" {
+            { ConvertFrom-Markdown -InputObject 1 -ErrorAction Stop } | Should -Throw -ErrorId 'InvalidInputObject,Microsoft.PowerShell.Commands.ConvertFromMarkdownCommand'
+        }
+    }
+
+    Context "Get/Set-MarkdownOption tests" {
+
+        BeforeAll {
+            $esc = [char]0x1b
+        }
+
+        BeforeEach {
+            $originalOptions = Get-MarkdownOption
+        }
+
+        AfterEach {
+            Set-MarkdownOption -InputObject $originalOptions
+        }
+
+        It "Verify default values for MarkdownOptions" {
+            $options = Get-MarkdownOption
+
+            $options.Header1 | Should -BeExactly "$esc[7m[7m$esc[0m"
+            $options.Header2 | Should -BeExactly "$esc[4;93m[4;93m$esc[0m"
+            $options.Header3 | Should -BeExactly "$esc[4;94m[4;94m$esc[0m"
+            $options.Header4 | Should -BeExactly "$esc[4;95m[4;95m$esc[0m"
+            $options.Header5 | Should -BeExactly "$esc[4;96m[4;96m$esc[0m"
+            $options.Header6 | Should -BeExactly "$esc[4;97m[4;97m$esc[0m"
+            $options.Code | Should -BeExactly "$esc[48;2;155;155;155;38;2;30;30;30m[48;2;155;155;155;38;2;30;30;30m$esc[0m"
+            $options.Link | Should -BeExactly "$esc[4;38;5;117m[4;38;5;117m$esc[0m"
+            $options.Image | Should -BeExactly "$esc[33m[33m$esc[0m"
+            $options.EmphasisBold | Should -BeExactly "$esc[1m[1m$esc[0m"
+            $options.EmphasisItalics | Should -BeExactly "$esc[36m[36m$esc[0m"
+        }
+
+        It "Verify Set-MarkdownOption can get options" {
+            Set-MarkdownOption `
+                -Header1Color "[4;1m" `
```

Comment:
ah - now I understand - you don't actually expect the user to give you the actual escape sequence.

Reply:
👍